src/Tizen.NUI/src/public/BaseHandle.cs

```diff
@@ -559,6 +566,24 @@ public class FocusRequestArgs : EventArgs
                 swigCPtr = value;
             }
         }
+
+        /// <summary>
+        /// swigCMemOwn
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected bool SwigCMemOwn
```

Comment:
I've modified to internal protected

Reply:
@huiyueun 
I think you need to add `[EditorBrowsable(EditorBrowsableState.Never)]` for `internal protected`. because derived calss can access `internal protected` types same as `protected` types.

Reply:
done

Reply:
Thank you~